.pre-commit-config.yaml

```diff
@@ -145,6 +145,14 @@ repos:
               echo;
               exit 1;
             } || true
+      - id: check-logger-usage
+        name: Check for proper logger usage
+        entry: python ./scripts/check-logger-usage.py
```

Comment:
instead of the script, how about just use grep like forbid-pytest-asyncio.

something like `grep -e "import logging" -e "from logging" | grep -v "allow-direct-logging`, then put `import logging  # allow-direct-logging` where appropriate